src/bindgen/utilities.rs

```diff
@@ -238,6 +261,25 @@ impl SynAttributeHelpers for [syn::Attribute] {
         })
     }
 
+    fn attr_name_value_lookup(&self, name: &str) -> Option<String> {
```

Comment:
Can we rewrite `has_attr_name_value` to share code with this somehow? Maybe making this return an `impl Iterator<Item = String>`, making the caller's duty to call `.next()` to reduce the lookup to one result?

Reply:
Done